doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -2121,15 +2120,15 @@ curl --header "Authorization: X-Road-apikey token=ff6f55a8-cc63-4e83-aa4c-55f99d
     "status": "saved
 ...
 ```
-
-The available APIs are documented in OpenAPI specification (TBD). Access rights for different APIs follow the same rules
-as the corresponding UI operations.
-Access to regular APIs is allowed from all IP addresses by default, but this can
+The available endpoints are documented in OpenAPI specification 
+([openapi-definition.yaml](https://github.com/nordic-institute/X-Road/blob/develop/src/proxy-ui-api/src/main/resources/openapi-definition.yaml)).
+Access rights for different endpoints follow the same rules as the corresponding UI operations.
+Access to regular endpoints is allowed from all IP addresses by default, but this can
```

Comment:
Not part of this task but just a thought: it would be nice to refer to an actual easy-to-read, graphical documentation of the API instead of just the yaml file. Maybe we could look into how to generate one with the OpenAPI generator that we are using.

Reply:
Agreed 100%. Maybe a new ticket about this.

Reply:
Slacked: https://niis.slack.com/archives/GGBHVCYSY/p1604386487056700